.codeclimate.yml

```diff
@@ -1,5 +1,50 @@
+version: "2"
+prepare:
+  fetch:
+    - url: https://raw.githubusercontent.com/getninjas/eslint-config-getninjas/master/common.js
+      path: .eslint/common.js
+    - url: https://raw.githubusercontent.com/getninjas/eslint-config-getninjas/master/index.js
+      path: .eslint/eslint-config-getninjas.js
+    - url: https://raw.githubusercontent.com/getninjas/dotfiles/master/.sass-lint.yml
+      path: .sass-lint-codeclimate.yml
 plugins:
   eslint:
     enabled: true
     config:
-      config: .eslintrc.js
+      config: .eslintrc-codeclimate.json
+  sass-lint:
+    enabled: true
+    config:
+      config: .sass-lint-codeclimate.yml
+checks:
+  duplication:
+    enabled: true
+    config:
+      languages:
+        - javascript
+  fixme:
```

Comment:
Isso é pra ele corrigir sozinho?

Reply:
No, it is for when leaving comment in the `FIXME: this is wrong` code, for example.